ProjectHive/ProjectHive.Services.NotificationsAPI/Extensions/AppBuilderExtensionStatusNotification.cs

```diff
@@ -0,0 +1,32 @@
+﻿using Microsoft.EntityFrameworkCore;
+using ProjectHive.Services.NotificationsAPI.Data;
+using ProjectHive.Services.NotificationsAPI.Data.Entities;
+
+namespace ProjectHive.Services.ProjectAPI.Extensions;
+
+public static class AppBuilderExtensionStatusNotification
+{
+    public static async Task<IApplicationBuilder> PrepareDatabaseNotification(this IApplicationBuilder app)
+    {
+        using var scopedServices = app.ApplicationServices.CreateScope();
+
+        var serviceProvider = scopedServices.ServiceProvider;
+        var dbContext = serviceProvider.GetRequiredService<ProjectHiveNotificationsDbContext>();
+
+        if (!await dbContext.NotificationStatuses.AnyAsync())
+        {
+            var status = new List<NotificationStatus>
+            {
+                new NotificationStatus { Name = "Sent" },
+                new NotificationStatus { Name = "Delivered" },
+                new NotificationStatus { Name = "Read" },
+                new NotificationStatus { Name = "Error" }
+            };
```

Comment:
Not sure that it is necessary to store those statues at the DB. Possible a slightly easier just use enum for it